lang/python/python3-readline/Makefile

```
@@ -0,0 +1,31 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-readline
+PKG_VERSION:=8.0.0
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=gnureadline
+PKG_HASH:=61eef72ed02dad415ede49752e972a1d2bd8c35c1e4464565d7effd806c99476
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=GPL-1.0-only
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-readline
+	SECTION:=lang
+	CATEGORY:=Languages
+	SUBMENU:=Python
+	TITLE:=Python3 readline
+	DEPENDS:=+python3 +libreadline +libncurses
+endef
```

```suggestion
define Package/python3-readline
  SECTION:=lang
  CATEGORY:=Languages
  SUBMENU:=Python
  TITLE:=Stand-alone GNU readline module
  DEPENDS:=+python3 +libreadline +libncurses
endef
```

Did you check your dependencies by using [this command](https://github.com/openwrt/packages/tree/master/lang/python#python-package-dependencies) from Python readme in this repository?

---

Not a stand-alone. It's compiled dynamically against system's libreadline. It says so in provided patch...

---

Replied here: https://github.com/openwrt/packages/pull/15369#discussion_r610089057

---

And answered soon after that.

---

Similar changes that were requested have been updated in the latest commit.